src/components/resultBox/resultBox.test.js

```diff
@@ -117,6 +117,48 @@ describe('Result Box', () => {
 
     wrapper = mount(<ResultBox {...props} />, options);
 
-    expect(wrapper).to.have.descendants('.add-follwed-account-button');
+    expect(wrapper).to.have.descendants('.add-to-bookmarks');
+  });
+
+  it('should go to next step after clicking "Retry"', () => {
+    props = {
+      copy: null,
+      title: 'test',
+      account: { hwInfo: {} },
+      body: 'test',
+      success: false,
+      recipientId: '123',
+      reset: () => {},
+      copyToClipboard: () => {},
+      t: () => {},
+      followedAccounts: ['123'],
+      prevStep: spy(),
+      transactionFailedClear: spy(),
+    };
+    wrapper = mount(<ResultBox {...props} />, options);
+    wrapper.debug();
+    wrapper.find('.add-to-bookmarks').at(0).simulate('click');
+    expect(props.prevStep).to.have.been.calledWith();
+    expect(props.transactionFailedClear).to.have.been.calledWith();
+  });
+
+  it('should go to next step after clicking "Add Bookmark"', () => {
+    props = {
+      copy: null,
+      title: 'test',
+      account: { hwInfo: {} },
+      body: 'test',
+      success: true,
+      recipientId: 455,
+      reset: () => {},
+      copyToClipboard: () => {},
+      t: () => {},
+      followedAccounts: ['123'],
+      nextStep: spy(),
+    };
+    wrapper = mount(<ResultBox {...props} />, options);
```

Comment:
It would be more readable to see what is the diff of `props` in this `it` from the previous one if you override only the props that should be different. And in a way that doesn't change `props` variable as described in another comment below.